Fix a couple of missing types

alexwlchan opened the pull request.

rcantin-w reviewed (dismissed):
👍

rcantin-w commented on common/data/hardcoded-ids.ts:
```diff
@@ -3,26 +3,12 @@
-  },
+export const collectionVenueIds = {
+  galleries: 'Wsttgx8AAJeSNmJ4',
+  libraries: 'Library',
```
Should this be `WsuS_R8AACS1Nwlx`?

alexwlchan replied:
yup, oops

rcantin-w commented:
<img width="226" alt="Screenshot 2023-01-16 at 09 48 29" src="https://user-images.githubusercontent.com/110461050/212649772-6f203e96-de4e-4fdf-9be1-31df788bea07.png">

The pending UI change is "Galleries" changing to "Galleries and reading room" which looks a bit off. I can't quite place why it's happening, though, as that data seems to be coming from Prismic. Maybe it was changed for a little bit, then changed again but Chromatic hasn't rerun?

alexwlchan commented:
It’s because we used to get the venue name from our hard-coded list, then I switched to get it from Prismic, which has the full title.

I’m tired of fighting this particular issue though, so I’ve removed it from this PR for now.

rcantin-w commented:
I'm sure we could figure out a workaround if you want to discuss it later on!
